plugins/single-instance/src/platform_impl/linux.rs

```diff
@@ -28,14 +31,27 @@ impl<R: Runtime> SingleInstanceDBus<R> {
     }
 }
 
+#[cfg(feature = "semver")]
+fn dbus_id(config: &Config, version: semver::Version) -> String {
+    let mut id = config.identifier.replace(['.', '-'], "_");
+    id.push('_');
+    id.push_str(semver_compat_string(version).as_str());
+    id
+}
```

Comment:
we need to do the same for macOS backend as well, no?

Reply:
now i regret adding support for macos...